backend/db/scripts/migrations/01-migration.sql

```diff
@@ -0,0 +1,20 @@
+CREATE DATABASE movie_matcher;
+
+
+CREATE TABLE sessions (
+    id varchar(255) UNIQUE NOT NULL PRIMARY KEY,
+    active boolean DEFAULT TRUE,
+    match_found boolean DEFAULT FALSE,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+); 
+
+
+CREATE TABLE session_codes (
+    id varchar(255) UNIQUE NOT NULL PRIMARY KEY,
+    access_code varchar(255) UNIQUE NOT NULL,
+    active boolean DEFAULT TRUE,
```

Comment:
No, they are separate. My thought process was to flip the session_code status based on if they're able to keep swiping. Eg. if they swiped 50 times, it would be inactive, and left inactive unless their partner also finished swiping and no match was found. In that case, then both would get an active status and more options to swipe on would be available. 

As I type this out, it seems a bit redundant. I could simply do the checks based on counts rather than manage a flag that may accidentally become out of sync.